libs/ledger-live-common/src/config/sharedConfig.ts

```diff
@@ -0,0 +1,78 @@
+import { ConfigSchema } from "@ledgerhq/live-config/LiveConfig";
+
+const liveConfig: ConfigSchema = {
+  cosmos: {
+    type: "object",
+    default: {
+      axelar: {
+        lcd: "https://axelar-api.polkachu.com",
+        minGasPrice: 0.07,
+        ledgerValidator: "axelarvaloper1fgklp9hemczlwtqp9jqzq3xahh38hznx7vd805",
+      },
+      cosmos: {
+        lcd: "https://cosmoshub4.coin.ledger.com",
+        ledgerValidator: "cosmosvaloper10wljxpl03053h9690apmyeakly3ylhejrucvtm",
+        minGasPrice: 0.025,
+      },
+      desmos: {
+        lcd: "https://desmos-api.ibs.team",
+        minGasPrice: 0.0025,
+      },
+      dydx: {
+        lcd: "https://api.dydx.nodestake.top:443",
+        minGasPrice: 12500000000,
+        ledgerValidator: "dydxvaloper1gffkd68xcnfpzcsplf0fsuetxkysunud6a900w",
+      },
+      nyx: {
+        lcd: "https://api.nyx.nodes.guru",
+        minGasPrice: 0,
+      },
+      onomy: {
+        lcd: "https://rest-mainnet.onomy.io",
+        minGasPrice: 0.003,
+        ledgerValidator: "onomyvaloper1fgklp9hemczlwtqp9jqzq3xahh38hznxu9mtmf",
+      },
+      osmo: {
+        lcd: "https://osmosis-api.polkachu.com",
+        ledgerValidator: "osmovaloper17cp6fxccqxrpj4zc00w2c7u6y0umc2jajsyc5t",
+        minGasPrice: 0.025,
+      },
+      persistence: {
+        lcd: "https://rest.core.persistence.one",
+        minGasPrice: 0.025,
+        ledgerValidator: "persistencevaloper1fgklp9hemczlwtqp9jqzq3xahh38hznxatty38",
+      },
+      quicksilver: {
+        lcd: "https://lcd.quicksilver.zone",
+        minGasPrice: 0.0025,
+        ledgerValidator: "quickvaloper1fgklp9hemczlwtqp9jqzq3xahh38hznx02n4pp",
+      },
+      secret_network: {
+        lcd: "https://lcd.secret.express",
+        minGasPrice: 0.25,
+      },
+      sei_network: {
+        lcd: "https://sei-api.polkachu.com",
+        minGasPrice: 0.1,
+      },
+      stargaze: {
+        lcd: "https://stargaze-api.polkachu.com",
+        minGasPrice: 1,
+      },
+      umee: {
+        lcd: "https://umee-api.polkachu.com",
+        minGasPrice: 0.1,
+      },
+      coreum: {
+        lcd: "https://full-node.mainnet-1.coreum.dev:1317",
+        minGasPrice: 0.1,
+      },
+      injective: {
```

Comment:
We miss a ledgerValidator here /!\

https://ledger-live-production-default-rtdb.europe-west1.firebasedatabase.app/config.json

Please check all of them

Reply:
good catch. In fact, I miss this config because the "SharedConfig" is a migration of the current codebase from: https://github.com/LedgerHQ/ledger-live/blob/24ebd63bf5c6b9534e4d6f103736f2bfc2d47df5/libs/ledger-live-common/src/config/defaultConfig.ts
I now add the missing injective config in my another PR: 
https://github.com/LedgerHQ/ledger-live/pull/5763/commits/3e4d8cdb7644737b0b5d4079e99e3320d9265561